src/components/landing-page/landing-page.controller.ts

```diff
@@ -88,10 +92,20 @@ export class LandingPageController implements angular.IController {
     }
 
     angular.element(this.$window).bind('resize', this.onWindowResize);
```

Comment:
We probably want to debounce calls to this since resize triggers a lot of events